TYPO3.Flow/Classes/TYPO3/Flow/Resource/ResourceManager.php

```diff
@@ -411,6 +413,39 @@ public function getPublicPackageResourceUri($packageKey, $relativePathAndFilenam
     }
 
     /**
+     * Returns the public URI for a static resource provided by the public package
+     *
+     * @param string $path
+     * @return string
+     * @throws Exception
+     * @api
+     */
+    public function getPublicPackageResourceUriByPath($path)
+    {
+        $this->initialize();
+        list($package, $path) = $this->getPackageAndPathByPublicPath($path);
```

Comment:
Agree with @aertmann - it's important that an API user knows exactly what to pass to that method and what will be returned. We have to be picky when its an API method, so please also add documentation to the params.